spec.md

```diff
@@ -209,6 +209,18 @@ that contains both context and data).
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
+### sampled-rate
+* Type: Positive Integer
+* Description: The rate at which this event has already been sampled. Represents
+  the number of similar events that happened but were not sent.
```

Comment:
I like this much better. One nit... its the # of events not sent, minus 1, right? Meaning, in your example below wouldn't the rate be 29, not 30 if this is the # of events NOT sent?  I'm counting the current event as "the one".

Reply:
@maplebed ^^